src/main/java/org/bihealth/mi/easybus/MessageFragment.java

```diff
@@ -0,0 +1,115 @@
+package org.bihealth.mi.easybus;
+
+import java.io.IOException;
+import java.util.Objects;
+
+/**
+ * Represents a fragment/splitted part of a { @link org.bihealth.mi.easybus.message message}
+ * 
+ * @author Felix Wirth
+ *
+ */
+public class MessageFragment extends Message {
+    
+    /** SVUID */
+    private static final long serialVersionUID = -14070272349211270L;
+    /** Number of this message fragment */
+    private final int         splitNr;
+    /** Total of fragments */
+    private final int         splitTotal;
+    /** Id of message (same over all fragments) */
+    private final String      id;
+    /** The fragment content */
+    private final String      content;
+    
+    /**
+     * Creates a new instance
+     * 
+     * @param id
+     * @param splitNr
+     * @param splitTotal
+     * @param content
+     */
+    public MessageFragment(String id, int splitNr, int splitTotal, String content) {
+        // Super
+        super(content);
+        
+        // Check
+        if(id == null || content == null) {
+            throw new IllegalArgumentException("Id and content can not be null!");
+        }
+        
+        if(splitNr >= splitTotal || splitTotal < 1) {
+            throw new IllegalArgumentException("Please provide a number >= 1 for splitTotal and a splitNr < splitTotal");
```

Comment:
Besser nicht direkt die Variablennamen nennen (können sich theoretisch ändern). Wie wäre "Number of fragments must be greater than 0 and fragment number must be lower than number of fragments"?